### ISSUE TITLE
NoMethodError on login

### ISSUE THREAD

**Issue by lebalz** [swissdrg/download-portal#9]
Reproduction:
1. create a new user as admin (Benutzer>Hinzufügen):
```
Benutzername: user
E-mail: user@test.ch
Name der Institution: foobar
Rolle: End User
```

2. logout
3. without activation, try to login with the new user and an empty or random password

![Screenshot from 2020-04-27 14-52-37](https://user-images.githubusercontent.com/22354689/80374316-c8968100-8896-11ea-804d-21d902a59821.png)

**Comment by rafibu** [comment]
Should already be fixed with commit ba746a8d rafibu <rafaelburkhalter@live.com> on 30.04.2020 at 15:31 "fixed Nullpointer where Login has no password"

**Comment by lebalz** [comment]
Ok perfect, then i close this issue 👍
